### DISCUSSION TITLE
z not adding to db in powershell

### DISCUSSION THREAD

**Discussion by snarum** [ajeetdsouza/zoxide#293] [Q&A] [answered]
Hi
I'm all new to Z and I can't get it to work as I would expect. I installed using Chocolatey and added the snippet to the end of my profile.ps1 file. when I start powershell there is no error messages, and I can navigate using z as I would expect. But there is no fast navigation, and when i type zoxide query --list there is no entries. Am I doing something wrong?

**Comment by snarum** [comment]
I'm using poshgit and oh-my-posh, so that might affect things? Scott Hanselman is also using those and it doesn't seem like he is experiencing the same problems.

**Reply by ajeetdsouza** [reply]
What version of PowerShell / zoxide are you using?

```sh
$ Get-Host | Select-Object Version
$ zoxide --version
```

**Reply by snarum** [reply]
zoxide v0.7.9, powershell 7.1.5. I don't know what i did, but it started working now :)

**Comment by ajeetdsouza** [comment, marked as answer]
Closing this. Feel free to post here if you figure out what the problem was.

**Comment by arthur-s** [comment]
I had same issue, but with strship (it's similar to git-posh)
I changed the order of initialisation, and it helped.
Keep zoxide initialization after git-posh, or at the bottom of Profile file.

**Reply by ajeetdsouza** [reply]
The README mentions that zoxide must be initialized at the end of your config for exactly this reason.

**Reply by psantosl** [reply]
Exaclty! Just living it at the end worked :-)

**Comment by GitMurf** [comment]
I similarly was having this issue, and the weird part is that it started AFTER v0.9.0. The solution indeed was making sure zoxide init was at the end of the powershell profile. The weird thing is though that previous to v0.9.1, it worked without being at the end of the profile 🤷‍♂️ That is what tripped me up as I posted here: https://github.com/ajeetdsouza/zoxide/issues/900
